### PR TITLE
Add email address for letsencrypt

### PR DESCRIPTION
**Pull request by muratugureminoglu**
https://github.com/ant-media/Ant-Media-Server/issues/3789

### CODE REVIEW

**Review by mekya — CHANGES_REQUESTED** [review]
Shall we make email parameter as optional?  Otherwise it breaks the compatibility and it will cause problem for the users. 

If email is present as parameter, we can use in certbot. 
If it's not present, we can call  with `--register-unsafely-without-email` as before

**Comment by mekya** [comment]
A kind reminder for this PR @muratugureminoglu

**Comment by muratugureminoglu** [comment]
> Shall we make email parameter as optional? Otherwise it breaks the compatibility and it will cause problem for the users.
> 
> If email is present as parameter, we can use in certbot. If it's not present, we can call with `--register-unsafely-without-email` as before

It 's done.

File: src/main/server/enable_ssl.sh [hunk 1]
```diff
@@ -20,15 +20,16 @@ do
...
    esac
 done
 
 ERROR_MESSAGE="There is a problem in installing SSL to Ant Media Server.\n Please take a look at the logs above and try to fix.\n If you do not have any idea, contact@antmedia.io"
 
 usage() {
   echo "Usage:"
-  echo "$0 -d {DOMAIN_NAME} [-i {INSTALL_DIRECTORY}]"
-  echo "$0 -d {DOMAIN_NAME} [-i {INSTALL_DIRECTORY}] [-v {route53 or custom}]"
+  echo "$0 -d {DOMAIN_NAME} [-i {INSTALL_DIRECTORY}] -e {YOUR_EMAIL}"
```
Comment (mekya) [inline src/main/server/enable_ssl.sh]: It seems that e-mail is optional. AFAIK, optional parameters are shown with square brackets

**Review by mekya — APPROVED** [review]
I've just changed the command line info and put the e-mail to the square brackets because e-mail is optional

**Pull request merged**